LeggedRobotCfg:
  init_state:
    # target angles [rad] when action = 0.0
    default_joint_angle:
      LF_HAA_joint: 0.1
      LF_HFE_joint: 0.8
      LF_KFE_joint: -1.5

      RF_HAA_joint: -0.1
      RF_HFE_joint: 0.8
      RF_KFE_joint: -1.5

      LH_HAA_joint: 0.1
      LH_HFE_joint: 1.0
      LH_KFE_joint: -1.5

      RH_HAA_joint: -0.1
      RH_HFE_joint: 1.0
      RH_KFE_joint: -1.5

  control:
    # PD Drive parameters:
    stiffness: 40.0
    damping: 1.0
    action_scale: 0.25
    decimation: 10

  normalization:
    clip_scales:
      clip_observations: 100.
      clip_actions: 1.2
    obs_scales:
      lin_vel: 2.0
      ang_vel: 0.25
      dof_pos: 1.0
      dof_vel: 0.05
      height_measurements: 5.0

  depth_image:
    original: [ 106, 60 ]
    resized: [ 87, 58 ]
    near_clip: 0.0
    far_clip: 2.0
    horizontal_fov: 87
    buffer_len: 2
  
  size:
    actions_size: 12
    observations_size: 753
    depth_latent_size: 32
    predicted_yaw_size: 2
    proprio_observation_size: 53
    proprio_history_len: 10
